best_practices.rst

```diff
@@ -326,7 +326,7 @@ Use the XLIFF Format for Your Translation Files
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Of all the translation formats supported by Symfony (PHP, Qt, ``.po``, ``.mo``,
-JSON, CSV, INI, etc.), ``XLIFF`` and ``gettext`` have the best support in the tools used
+JSON, CSV, INI), ``XLIFF`` and ``gettext`` have the best support in the tools used
```

Comment:
I think this must be reverted, it's not a complete list and shouldn't start with "e.g." either

Reply:
Thanks! I've been greedy there